audresample/core/api.py

```diff
@@ -175,8 +175,8 @@ def resample(
 ) -> np.ndarray:
     r"""Resample signal to a new sampling rate.
 
-    The returned signal is always of type ``np.float32``
-    with shape (``channels``, ``samples``).
+    Supports only single precision signals (e.g. datatype float32).
```

Comment:
Why "e.g."?

Reply:
Because I'm not an expert on this topic, and I understood https://numpy.org/doc/stable/reference/arrays.scalars.html#numpy.single the way that it is only called `float32` on unix systems?

Reply:
But now it sounds like other datatypes are supported. Maybe we also rephrase to "signals in single precision floating point format"?

Reply:
Changed it accordingly.